### PR TITLE
additional refactoring for unit tests

### PR DESCRIPTION
**Pull request by fedordavid**

### CODE REVIEW

File: freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+        public CustomerViewBuilder(CustomerViewBuilder customer)
+        {
+            this.Id = customer.Id;
+            this.Name = customer.Name;
+            this.Email = customer.Email;
+            this.Phone = customer.Phone;
+        }
+
+        public CustomerViewBuilder()
```
Comment (TomasDurica) [inline freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs:21]: Ziadne defaults?

File: freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        }
+
+        public CustomerViewBuilder WithId(string id)
+        {
+            var result = new CustomerViewBuilder(this)
+            {
+                Id = new Guid(id)
+            };
+
+            return result;
```
Comment (TomasDurica) [inline freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs]: mozes dat rovno return new ....

File: freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+using System;
+
+namespace Freddy.Application.UnitTests.API.Controllers.CustomerController
+{
+    public class CustomerViewBuilder
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string Email { get; set; }
+        public string Phone { get; set; }
```
Comment (TomasDurica) [inline freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs]: Mas dovod aby to bolo public?

**Review by TomasDurica — CHANGES_REQUESTED** [review]

File: freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs (hunk 2 above)
Comment (fedordavid) [reply freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs]: ok

File: freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs (hunk 3 above)
Comment (fedordavid) [reply freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs]: z vonku sa properties nebudu pouzivat len cez With_whatever takze nie, opravim

**Review by TomasDurica — APPROVED** [review]

**Pull request merged**